app/script/tracking/EventTrackingRepository.js

```diff
@@ -80,181 +77,144 @@ z.tracking.EventTrackingRepository = class EventTrackingRepository {
       }
     }
   }
 
-  _subscribe_to_events() {
-    amplify.subscribe(z.event.WebApp.ANALYTICS.CLOSE_SESSION, this.close_session.bind(this));
-    amplify.subscribe(z.event.WebApp.ANALYTICS.CUSTOM_DIMENSION, this.set_custom_dimension.bind(this));
-    amplify.subscribe(z.event.WebApp.ANALYTICS.EVENT, this.tag_event.bind(this));
-    amplify.subscribe(z.event.WebApp.ANALYTICS.START_SESSION, this.start_session.bind(this));
+  _subscribe_to_tracking_events() {
+    amplify.subscribe(z.event.WebApp.ANALYTICS.CUSTOM_DIMENSION, this, (...args) => {
+      if (this.is_user_tracking_activated) {
+        this._set_super_property(...args);
+      }
+    });
+
+    amplify.subscribe(z.event.WebApp.ANALYTICS.EVENT, this, (...args) => {
+      if (this.is_user_tracking_activated) {
+        this._track_event(...args);
+      }
+    });
   }
 
-  _unsubscribe_from_events() {
-    amplify.unsubscribeAll(z.event.WebApp.ANALYTICS.CLOSE_SESSION);
+  _unsubscribe_from_tracking_events() {
     amplify.unsubscribeAll(z.event.WebApp.ANALYTICS.CUSTOM_DIMENSION);
     amplify.unsubscribeAll(z.event.WebApp.ANALYTICS.EVENT);
-    amplify.unsubscribeAll(z.event.WebApp.ANALYTICS.START_SESSION);
   }
 
-
-  //##############################################################################
-  // Localytics
-  //##############################################################################
-
-  close_session() {
-    if (this.localytics && this.privacy_preference) {
-      this.logger.info('Closing Localytics session');
-
-      if (this.session_interval) {
-        window.clearInterval(this.session_interval);
-        this.session_interval = undefined;
-      }
-
-      this.localytics('upload');
-      this.localytics('close');
-    }
+  _set_super_property(super_property, value) {
+    this.logger.info(`Set super property '${super_property}' to value '${value}'`);
+    const super_properties = {};
+    super_properties[super_property] = value;
+    this.mixpanel.register(super_properties);
   }
 
-  set_custom_dimension(custom_dimension, value) {
-    if (this.localytics) {
-      this.logger.info(`Set Localytics custom dimension '${custom_dimension}' to value '${value}'`);
-      this.localytics('setCustomDimension', custom_dimension, value);
+  _track_event(event_name, attributes) {
+    if (attributes) {
+      this.logger.info(`Tracking event '${event_name}' with attributes: ${JSON.stringify(attributes)}`);
+    } else {
+      this.logger.info(`Tracking event '${event_name}' without attributes`);
     }
-  }
 
-  start_session() {
-    if (this.privacy_preference && !this.session_interval) {
-      if (!this.localytics) {
-        this._init_localytics();
-      }
+    // During the transition phase (Localytics -> Mixpanel), we only want to log certain events:
+    const allowed_events = [
+      z.tracking.EventName.MEDIA.COMPLETED_MEDIA_ACTION,
+      z.tracking.EventName.TRACKING.OPT_IN,
+      z.tracking.EventName.TRACKING.OPT_OUT,
+    ];
```

Comment:
Let's define this in config up top like in other repositories instead of creating a variable each time we call `_track_event`